### ISSUE TITLE
'Total Character Skillpoints' only shows main character skillpoints

### ISSUE THREAD

**Issue by yarmout** [eveseat/seat#673]
* **Problem**: 'Total Character Skillpoints' on the home dashboard only displays the skillpoints of the user's main character
* **Expected**: 'Total Character Skillpoints' to display the total skillpoints of all the user's characters
* **Logs / Screenshots / Proof**: Skillpoints matches the skillpoints of whichever character is selected as the main character
![Character 1](https://i.imgur.com/k6hLaNF.png)
![Character 2](https://i.imgur.com/Yswkai9.png)

* **Version Info**: 
OS: Debian 10
Install method: Docker
Version info: 
![Version info](https://i.imgur.com/Wi6PteT.png)

**Comment by Crecket** [comment]
TLDR, the three other stats on the homepage allow for multiple character IDs, whereas the total SP stat only allow for 1 character ID which in this case is set to the current character ID. See:

https://github.com/eveseat/web/blob/7a82b36850a2cddc4ed0db852618aa2aa61dacc5/src/Http/Controllers/HomeController.php#L61

And then underneath

https://github.com/eveseat/web/blob/7a82b36850a2cddc4ed0db852618aa2aa61dacc5/src/Traits/Stats.php#L70-L75

I'm not familiar with the syntax but I assume we could just refactor it to be something like  this as done with the character wallet/mining values?

```PHP
public function getTotalCharacterSkillpoints(array $character_ids): ?float
{
     return CharacterInfoSkill::whereIn('character_id', $character_ids)
        ->sum('total_sp');
}
```

**Comment by warlof** [comment]
Hi,

This metrics has indead been refactored with SeAT 4.
While working on it, I figure summing ISK, Mining, counting killmails and characters make sense.

However, summing skillpoints don't really mean anything.
As a result, I swap it to the chosed main character, assuming it might be the one user prefer to track.

That issue put the question back on the table :
Should we display sum of SP from all owned characters by an user, or only its main one ?!

**Comment by yarmout** [comment]
I think they are a few reasons why displaying total SP would be a good idea.
1) It's already possible to view individual character SP on the character pages.
2) Viewing total SP offers something unique. Many of my users enjoyed seeing how much total SP they have accumulated across all of their characters.
3) It's more consistent with the other information displayed on the home dashboard, which provides a summary of all of your characters, not just the selected main.

For those reasons I'd be in favor of keeping it as total skill points across all characters

**Comment by Crecket** [comment]
If it isn't showing "total" skill points, it should either be moved or relabled in my opinion. Especially considering that the previous Seat versions did add up all the skill points for the different characters and now it doesn't even though it uses the same location and label.

**Comment by warlof** [comment]
This has been addressed by eveseat/web@4.2.3 and seat@4.0.22
